src/components/Proposals/ProposalCreation/SubmitButton.tsx

```diff
@@ -78,26 +86,47 @@ export default function SubmitButton({
   }, []);
 
   return (
-    <Button
-      type="submit"
-      variant={"outline"}
-      disabled={isLoading || onPrepareError}
-      className={cx(["w-[40%]", onPrepareError && styles.submit_button])}
-      onClick={(e) => {
-        e.preventDefault();
-        if (!isConnected) {
-          setOpen(true);
-          return;
-        }
-        if (formTarget.current?.checkValidity() && !onPrepareError) {
-          formTarget.current?.reportValidity();
-          submitProposal();
-        }
-      }}
-    >
-      {/* hack to suppress Suspense boundary error */}
-      {isClient && isConnected ? "Submit proposal" : "Connect wallet"}
-    </Button>
+    <>
+      <div className="flex flex-col gap-2">
+        {!!inputDataError && (
+          <p className="text-red-700 text-sm max-w-[420px] break-words">
+            {(inputDataError as { message?: string })?.message ||
+              JSON.stringify(inputDataError)}
```

Comment:
Getting this loud error when connecting with a wrong wallet. We might wanna make it less loud and just show `Only the Optimism Foundation manager address can create proposals for the time being.` in this case
<img width="1469" alt="Screenshot 2024-02-15 at 4 14 23 PM" src="https://github.com/voteagora/agora-next/assets/20438019/656c9895-9ec4-4d62-9637-ee10522272c1">